_data/settings/components/table.yml

```diff
@@ -35,6 +40,11 @@ contents:
     var: $theme-table-sorted-stripe-background-color
     default: '"blue-cool-10v"'
     type: color
+  - name: Sticky header top offset
+    description: Defines the top offset for sticky headers.
+    var: $theme-table-sticky-top-offset
+    default: '-1px'
+    type:
```

Comment:
> **Note**
> This does not have an assigned type because it accepts any custom value, not just `units`.

Reply:
Should we specify it takes number values or is the implication of the default value and name enough?